src/main/resources/org/jenkinsci/plugins/plumber/PlumberInterpreter.groovy

```diff
@@ -47,18 +47,7 @@ class PlumberInterpreter implements Serializable {
     }
 
     def fromYaml(String yamlFile) {
-        def yamlText
-
-        // Need to run on some arbitrary node to read the file.
-        // TODO: Find a way to just read that file from the parent flyweight.
```

Comment:
When [JENKINS-33273](https://issues.jenkins-ci.org/browse/JENKINS-33273) is implemented, you should get this optimization for free, at least in multibranch projects.